WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2721,10 +2721,21 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                     }
                     break;
                 case RequestCodes.FILE_LIBRARY:
-                    uris = WPMediaUtils.retrieveMediaUris(data);
-                    mAnalyticsTrackerWrapper.track(Stat.EDITOR_ADDED_FILE_VIA_LIBRARY);
-                    for (Uri item : uris) {
-                        mEditorMedia.addNewMediaToEditorAsync(item, false);
+                    if (mConsolidatedMediaPickerFeatureConfig.isEnabled()) {
```

Comment:
I'm wondering if it might make sense to reuse the existing `PHOTO_PICKER` request code instead of copying the same logic here.